inc/classes/subscriber/Optimization/class-scripts-escaping-subscriber.php

```diff
@@ -0,0 +1,75 @@
+<?php
+namespace WP_Rocket\Subscriber\Optimization;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Handles escaping of closing HTML tags in inline scripts to prevent them from being removed by DOMDocument
+ *
+ * @since 3.1
+ * @author Remy Perona
+ */
+class Scripts_Escaping_Subscriber implements Subscriber_Interface {
+	/**
+	 * @inheritDoc
+	 */
+	public static function get_subscribed_events() {
+		/** This action is documented in inc/classes/subscriber/class-google-tracking-cache-busting-subscriber.php */
+		if ( apply_filters( 'rocket_buffer_enable', true ) ) {
+			return [
+				'rocket_buffer' => [
+					[ 'escape_html_in_scripts', 1 ],
```

Comment:
I guess the number is a hook priority.
In case you're not aware of it, and absolutely need to be before everybody, we can use negative values.

Reply:
yes it's the hook priority currently the lowest priority used elsewhere is 10 so 1 is fine